requirements.txt

```diff
@@ -1,41 +1,43 @@
 pip==9.0.3
 Django==1.11.20 # rq.filter: <1.12
--e git+https://github.com/OpenDataServices/flatten-tool.git@v0.5.0#egg=flattentool
--e git+https://github.com/open-contracting/lib-cove-ocds.git@v0.2.1#egg=libcoveocds
+-e git+https://github.com/OpenDataServices/flatten-tool.git@4c13ef0b32a59e810919a3de09bc8f64ce8f9392#egg=flattentool
+-e git+https://github.com/open-contracting/lib-cove-ocds.git@74c459c06136af45db76487f39408664fd2d4854#egg=libcoveocds
 -e git+https://github.com/OpenDataServices/lib-cove-web.git@1489d3d60296399d768dcd81b20fec788f792487#egg=libcoveweb
--e git+https://github.com/OpenDataServices/lib-cove.git@2fee8ee2e572d26415c503e05d79b42ad95dc25d#egg=libcove
-django-bootstrap3==9.1.0
-django-debug-toolbar==1.9.1
+-e git+https://github.com/OpenDataServices/lib-cove.git@13b6ecb76a00db6bb0fdbfa28e5672045c6c9eec#egg=libcove
+django-bootstrap3==11.0.0
+django-debug-toolbar==1.11
 requests==2.21.0
-cached-property==1.4.0
+cached-property==1.5.1
 dealer==2.0.5
-django-environ==0.4.4
-jsonschema==2.6.0
+django-environ==0.4.5
+jsonschema==3.0.1
 json-merge-patch==0.2
-python-dateutil==2.7.3
-raven==6.6.0
+python-dateutil==2.8.0
+raven==6.10.0
 strict-rfc3339==0.7
-rfc3987==1.3.7
+rfc3987==1.3.8
 rfc6266==0.0.4
-CommonMark==0.7.4
-bleach==2.1.3
-xmltodict==0.11.0
+bleach==3.1.0
+xmltodict==0.12.0
+rangedict==0.1.6
 ## The following requirements were added by pip freeze:
-certifi==2018.1.18
+attrs==19.1.0
+certifi==2018.11.29
 chardet==3.0.4
+commonmark==0.8.1
+contextlib2==0.5.5
 et-xmlfile==1.0.1
-future==0.16.0
-html5lib==1.0.1
-idna==2.6
-jdcal==1.3
-jsonref==0.1
+future==0.17.1
+idna==2.8
+jdcal==1.4
+jsonref==0.2
 LEPL==5.1.3
-lxml==4.3.0
-openpyxl==2.5.5
-
-pytz==2018.3
-schema==0.6.7
-six==1.11.0
+lxml==4.3.2
+openpyxl==2.6.1
```

Comment:
We're not user if openpyxl 2.6 works with flatten-tool properly - https://github.com/OpenDataServices/flatten-tool/issues/287 (although, it might just be the tests that need updating).

Reply:
I have changed the version to the original one